_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -29093,68 +29025,81 @@
 	},
 /turf/open/floor/plasteel,
 /area/hallway/primary/central)
-"bdy" = (
-/obj/structure/cable/yellow{
-	icon_state = "1-2"
-	},
-/obj/structure/disposalpipe/sorting/mail{
-	dir = 1;
-	sortType = 22
+"bdA" = (
+/obj/effect/decal/cleanable/dirt,
+/turf/closed/wall,
+/area/security/vacantcommissary)
```

Comment:
Noticed you were using the security path and this has some minor issues because some events and stuff look for it in specific. Repathing it in #41017

Reply:
yeah I was just following the way other vacant rooms did it, do you need me to do something on my end?